nautobot_ssot/jobs/examples.py

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
While I see that these are necessary changes, I'm just wondering why they are introduced in this PR?

Reply:
Also the implementation of this example job is swapped over to the `contrib` base models in `develop`, so this may unfortunately need to be redone once more when we get the latest changes from develop into the 2.0 stream

Reply:
@Kircheneer the amount of work to refactor the example job so it is 2.0 compatible is out of scope at this time. We can re-introduce this later if we decide it's needed.